<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="hal-51" name="Default test(failed)(failed)" tests="1" failures="1" timestamp="10 Aug 2013 08:46:48 GMT" time="5.903" errors="0">
  <testcase name="addBundle" time="0.113" classname="com.bleulace.domain.management.domain.ProjectCommandTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.bleulace.domain.management.domain.ProjectCommandTest.addTask(ProjectCommandTest.java:77)
	at com.bleulace.domain.management.domain.ProjectCommandTest.addBundle(ProjectCommandTest.java:68)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:155)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- addBundle -->
</testsuite> <!-- Default test(failed)(failed) -->
